docs/testplan/latency_test.md

```diff
@@ -0,0 +1,99 @@
+# SONiC Switch Latency Test
+
+- [SONiC Switch Latency Test](#sonic-switch-latency-test)
+  - [Test Objective](#test-objective)
+  - [Test Setup](#test-setup)
+  - [Test Case 1: Full-mesh Latency Test](#test-case-1-full-mesh-latency-test)
+  - [Test case 2: Latency Test with Oversubscribed Port](#test-case-2-latency-test-with-oversubscribed-port)
+  - [Metrics Processing](#metrics-processing)
+
+## Test Objective
+
+This test aims to measure the latency introduced by a switch under the fully loaded condition.
+
+## Test Setup
+
+> **Notice**: Currently, we are testing exclusively in store-and-forward mode.
+
+1. **Maximize Traffic Load on DUT**
+
+   This test uses all traffic generator ports defined in the test topology to generate the maximum possible traffic volume, stressing the DUT to capture its latency behavior under full load.
+
+   - **Switch wiring**: All DUT ports must be connected to traffic generators—either directly or through intermediary devices.
+
+2. **Establish BGP Sessions and Routes**
+
+   Follow the steps outlined in the multi-tier BGP test plan to:
+   - Set up the necessary IPv6 BGP sessions
+   - Advertise the desired IPv6 routes
```

Comment:
we don't want to limit ourselves to ipv6 only. ipv6 is only a test variation, same test can happen on ipv4 as well.